bedrock/base/templates/includes/protocol/site-footer.html

```diff
@@ -46,7 +46,7 @@ <h5 class="mzp-c-footer-heading">
       </ul>
       <div class="mzp-c-footer-legal">
         <p class="mzp-c-footer-license">
-        {% if LANG.startswith('en-') %}
+        {% if LANG.startswith('en-') or LANG.startswith('es-') or LANG in ['de', 'fr', 'pl', 'it', 'zh-CN', 'zh-TW', 'ja', 'ko', 'ru'] %}
```

Comment:
Looks like `pt-BR` didn't make it into this list but everything else works as expected.

Reply:
Doh, I can add it.